app/scripts/services/dimFarmingService.factory.js

```diff
@@ -168,24 +181,23 @@
           var consolidateHashes = [
```

Comment:
Can't we make these user-defined, with a drag & drop, that persist. Like loadout builder has for locking slots.